resources/quality/xyzprinting/anti_bact/xyzprinting_da_vinci_jr_pro_xeplus_copper_0.40_antibact_coarse.inst.cfg

```diff
@@ -0,0 +1,29 @@
+[general]
+version = 4
+name = Coarse Quality
+definition = xyzprinting_da_vinci_jr_pro_xeplus
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = coarse
+material = xyzprinting_antibact_pla
```

Comment:
You didn't add that material, so this will never become active.

Reply:
![image](https://user-images.githubusercontent.com/7088747/131647678-9d0efcda-a739-40bc-ab62-255e19fcf912.png)

But i have added them in material folder.
Should i add other setting for them??

Reply:
OK,the pull request of the fdm_materials is on-going,thank you :)